submarine-workbench/workbench-web/src/app/pages/workbench/experiment/experiment-home/experiment-home.component.ts

```diff
@@ -94,9 +94,11 @@ export class ExperimentHomeComponent implements OnInit {
             item.duration = this.experimentService.durationHandle(result);
           }
         });
-        this.checkedList = [];
-        for (let i = 0; i < this.experimentList.length; i++) {
-          this.checkedList.push(false);
+        if(!isRefetch){
```

Comment:
Add a comment here
// If it is auto-reloading, we do not want to change the state of checkbox.

Reply:
Got it.